conans/client/remote_manager.py

```diff
@@ -99,12 +101,18 @@ def get_recipe(self, ref, remote):
         self._hook_manager.execute("pre_download_recipe", reference=ref, remote=remote)
 
         ref = self._resolve_latest_ref(ref, remote)
-        layout = self._cache.create_ref_layout(ref)
+        try:
+            layout = self._cache.ref_layout(ref)
+        except ConanReferenceDoesNotExist:
+            layout = self._cache.create_ref_layout(ref)
+
         layout.export_remove()
 
         t1 = time.time()
         download_export = layout.download_export()
         zipped_files = self._call_remote(remote, "get_recipe", ref, download_export)
+        remote_revisions = self._call_remote(remote, "get_recipe_revisions", ref)
+        ref_time = from_iso8601_to_timestamp(remote_revisions[0].get("time"))
```

Comment:
It seems the server is returning an iso date instead of the raw timestamp? Looks like something to change in the server API?

Reply:
Maybe at least also hide it behind the remote call (implement it in the rest api)?